### ISSUE TITLE
DC make file or how to build

### ISSUE THREAD

**Issue by ianmicheal** [Kannagi/LMP3D#3]
HI just found your project :) but i dont see how to build it or makefile or anything at all.. I like to test it and try optimize parts

**Comment by drogeek** [comment]
I think you mean `bin/bin2c/Makefile`

**Comment by Kannagi** [comment]
> HI just found your project :) but i dont see how to build it or makefile or anything at all.. I like to test it and try optimize parts

Yes sorry, there is only Code::Block project, I can help you, to compile it yourself, it is not very complicated (and requires little dependency)
